pkgs/by-name/bi/bisq2/package.nix

```diff
@@ -58,7 +58,7 @@ stdenvNoCC.mkDerivation rec {
 
```

Comment:
The publicKey above expired on 10. February, hence signature verification failed.